platformio.ini

```diff
@@ -247,10 +248,11 @@ lib_deps =
 [esp32c3]
 ;; generic definitions for all ESP32-C3 boards
 platform = espressif32@5.2.0
+platform_packages = 
 build_flags = -g
   -DARDUINO_ARCH_ESP32
   -DARDUINO_ARCH_ESP32C3
-  -DCONFIG_IDF_TARGET_ESP32C3
+  -DCONFIG_IDF_TARGET_ESP32C3=1
```

Comment:
Also never had issues with this.
But there is a difference if you add a space between `-D` and `CONFIG_...`.

Reply:
True, it's a very subtle difference. I just found out recently that the official ESP-IDF way is  `-DCONFIG_IDF_TARGET_ESP32C3=1`